docs/get-started/product-logo-library.njk

```diff
@@ -0,0 +1,136 @@
+---
+layout: layout-basic.njk
+title: Product logo library
+order: 4
+tags:
+  - getstarted
+---
+
+<!-- Body -->
+
+{% import 'component/get-started.njk' as getstarted %}
+
+{% call getstarted.section("Introduction") -%}
+
+  <p>Our product logos connect every new product or service to our parent brand, Red Hat. Their consistent format allows us to add new product logos to our portfolio without additional design or legal costs. To see our repo of logos, visit <a href="https://docs.google.com/presentation/d/1mrQd094Q7kse_rDb1BpcO7RDShsauoaZjxvEf6eW1cI/edit">this page</a>.</p>
+
+{%- endcall %}
+
+{% call getstarted.section("Using libraries") -%}
+
+  <p>Our libraries require access to Adobe Creative Cloud through a Red Hat account. These libraries are intended for designers who regularly use Adobe CC apps as part of their role at Red Hat. Access will not be granted to accounts that are not part of the Red Hat domain.</p>
+
+  <rh-cta variant="secondary">
+    <a href="https://url.corp.redhat.com/adobecc">Request a Creative Cloud license</a>
+  </rh-cta>
+
+{%- endcall %}
+
+{% call getstarted.section("Request access") -%}
+
+  <p>To request access to the Adobe CC libraries containing brand assets, fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSeZbyEh0CGJrujIocHuR4ZMWh_uew-IeUNWewBOb3GBnUHTrA/viewform?usp=sf_link">this form</a>. Please allow a week or less to receive access. Once we accept your request, you will receive invites to join the <strong>three libraries</strong> below. These libraries will then sync to your Adobe CC design programs and you can view the assets in the <em>Libraries</em> panel.</p>
```

Comment:
The titles of the three libraries should be listed out as text, instead of just being shown as an image.